.snippets/code/store-and-retrieve-data/use-storagehub-sdk/end-to-end-storage-workflow/output-01.html

```diff
@@ -0,0 +1,23 @@
+<div class="termynal" data-termynal>
+    <span data-ty="input"><span class="file-path"></span>ts-node index.ts</span>
+    <span data-ty><pre>Checking for bucket in MSP backend, attempt 1 of 10...
+Bucket not found in MSP backend yet (404).
+Checking for bucket in MSP backend, attempt 2 of 10...
+Bucket not found in MSP backend yet (404).
+Checking for bucket in MSP backend, attempt 3 of 10...
+Bucket not found in MSP backend yet (404).
+Checking for bucket in MSP backend, attempt 4 of 10...
+Bucket not found in MSP backend yet (404).
+Checking for bucket in MSP backend, attempt 5 of 10...
+Bucket not found in MSP backend yet (404).
+Checking for bucket in MSP backend, attempt 6 of 10...
+Bucket found in MSP backend: {
+  bucketId: '0x750337cba34ddcfdec3101cf8cc5ae09042a921b5571971533af2aab372604b9',
+  name: 'init-bucket',
+  root: '0x0000000000000000000000000000000000000000000000000000000000000000',
+  isPublic: true,
+  sizeBytes: 0,
+  valuePropId: '0x628a23c7aa64902e13f63ffdd0725e07723745f84cabda048d901020d200da1e',
+  fileCount: 0
+}</pre></span>
```

Comment:
```suggestion
}</pre></span>
<span data-ty="input"><span class="file-path"></span></span>
```
Adds the finishing line to indicate the end of output (blank command line) per style guide